helm/sematic-server/values.yaml

```diff
@@ -75,6 +75,45 @@ image:
   pull_policy: IfNotPresent
   #pull_secrets: []
 
+ray:
+  enabled: false
+  supports_gpus: false
+
+  # The Kubernetes node selector that will be used for Ray nodes
+  # that use GPUs. Value should be json encoded into a string.
+  gpu_node_selector: {}
+  #gpu_node_selector: {"node.kubernetes.io/instance-type": "g4dn.xlarge"}
+  
+  # The Kubernetes node selector that will be used for Ray nodes
+  # that don't use GPUs.
+  non_gpu_node_selector: {}
+  #non_gpu_node_selector: {"node.kubernetes.io/instance-type": "t3.large"}
+  
+  # The Kubernetes tolerations that will be used for Ray nodes
+  # that use GPUs.
+  gpu_tolerations: []
+  #gpu_tolerations:
+  #  - key: "nvidia.com/gpu"
+  #    operator: Equal
+  #    effect: NoSchedule
+  #    value: "true"
+  
+  # The Kubernetes tolerations that will be used for Ray nodes
+  # that don't use GPUs.
+  non_gpu_tolerations: []
```

Comment:
same as above

Reply:
this is just being consistent with the plugin's settings vars (already merged [here](https://github.com/sematic-ai/sematic/blob/main/sematic/plugins/kuberay_wrapper/standard.py))